### PR TITLE
REF/CLN: test_get_dummies

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/reshape/test_get_dummies.py [hunk 1]
```diff
@@ -223,12 +223,12 @@ def test_dataframe_dummies_prefix_list(self, df, sparse):
...
         expected = expected[["C"] + cols]
 
-        typ = SparseArray if sparse else pd.Series
+        typ = SparseArray if sparse else Series
         expected[cols] = expected[cols].apply(lambda x: typ(x))
         tm.assert_frame_equal(result, expected)
 
     def test_dataframe_dummies_prefix_str(self, df, sparse):
-        # not that you should do this...
+        # not that you sheould do this...
```
Comment (jschendel) [inline pandas/tests/reshape/test_get_dummies.py]: typo

**Comment by jreback** [comment]
lgtm. merge on green.

**Review by jschendel — APPROVED** [review]
lgtm, one small comment

**Review by datapythonista — APPROVED** [review]
Is there an ongoing effort to replace `pd.DataFrame` by `DataFrame` in our code and to be redundant with the test method names (`TestGetDummmies.test_basic` by `TestGetDummmies.test_get_dummies_basic`)?

Replacing loops by parametrization improves readability, but these other changes look very personal and opinionated, and if we want to make them a standard, would be nice to have an issue referenced here. Otherwise we can end up in a situation where every time someone works with a file, we'll get some `pd.` added and removed (for example, I would personally prefer to always have the module name and not import objects).

**Comment by jbrockmendel** [comment]
> and to be redundant with the test method names (TestGetDummmies.test_basic by TestGetDummmies.test_get_dummies_basic)?

In this case it is redundant, but in many other cases we'll see something like `test_basic` in a class without an informative name. It's also not uncommon to see tests for something other than get_dummies living inside a TestGetDummies class.  I think being explicit is worthwhile here.

> Is there an ongoing effort to replace pd.DataFrame by DataFrame in our code

Not that I'm aware of.  For files testing lower-level code, I like to have all the relevant imports at the top so that it is easy to tell what is tested in this file.  If `pd` is in the namespace, I can't rule anything out.  Not really relevant for this file, just a habit I've gotten into.

**Comment by jbrockmendel** [comment]
updated with typo fixup + green

**Comment by WillAyd** [comment]
> (for example, I would personally prefer to always have the module name and not import objects).

Yea @jorisvandenbossche and I agree, but at the same time I think someone needs to champion that cause if we want to establish a standard

**Comment by WillAyd** [comment]
Thanks @jbrockmendel

**Comment by jreback** [comment]
> > (for example, I would personally prefer to always have the module name and not import objects).
> 
> 
> 
> Yea @jorisvandenbossche and I agree, but at the same time I think someone needs to champion that cause if we want to establish a standard
> 
> 

i have always been in favor of this :)

moving to this will require a good code check though - s may need be that easy

**Comment by jorisvandenbossche** [comment]
IMO this doesn't necessarily require a code check initially. 
But we could already just stop merging PRs that do changes like `pd.DataFrame` -> `DataFrame` (or ask them to revert that part). @WillAyd gave the comment like https://github.com/pandas-dev/pandas/pull/33184#issuecomment-606957773 on several PRs recently I think, but in the end we always just merged the PR.

**Comment by jreback** [comment]
I care most about consistency in a single file meaning we shouldn’t mix ways of referencing common things like Series vs pd.Series

I do prefer using import of Series and not pd.Series

**Comment by jorisvandenbossche** [comment]
> I do prefer using import of Series and not pd.Series

I am confused now. I interpreted the above "i have always been in favor of this" as the opposite (https://github.com/pandas-dev/pandas/pull/33184#issuecomment-606961547), since that is replying to a quote of Will that is basically saying that he, Marc and I prefer to not import objects but modules (so using `pd.Series` instead of `Series`)

**Comment by jorisvandenbossche** [comment]
>  I think someone needs to champion that cause if we want to establish a standard

I opened a separate issue about it: https://github.com/pandas-dev/pandas/issues/33203. Let's try to decide there which of the two ways has our preference.

**Pull request merged**